src/app.js

```diff
@@ -15,18 +20,21 @@ const createTemplateByType = (type, createElement) => {
 export default {
     name: "InputFormats",
     components: {
-        VueMoney: () => import("./components/VueMoney"),
+        VueMoney: () => import("./components/VueMoney.js"),
     },
     props: {
-        type: {
-            type: String,
-            default: TemplateType.MONEY,
-        },
+        type: { type: String, default: TemplateType.MONEY },
+        value: { required: true, type: [Number, String], default: 0 },
```

Comment:
I suggest using this style of writing props (inline). 
I think it will be convenient.